docs/prerequisites/PREREQUISITES-LOCALLY.md

```diff
@@ -2,7 +2,7 @@
 
   1. [Create a Cloud SQL Instance](#create-cloud-sql-instance)
   1. [Install the protoc compiler](#install-the-protoc-compiler)
-  1. [Install mysql](#install-mysql-compiler)
+  1. [Install `mysql_config`](#install-mysql-compiler)
```

Comment:
[Install `mysql_config`](#install-mysql-config)

I think -- not sure if it's #install-mysql_config